tasks/common_packages.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Write the apt sources file
+  template:
+    src: sources.list.j2
+    dest: /etc/apt/sources.list
+    mode: 0644
+    owner: root
+    group: root
+
+
```

Comment:
-

Reply:
@dasJ where is the problem here?

Reply:
I think he is denouncing the unnecessary whitespace at the end of this file.